pkg/apis/kops/validation/validation.go

```diff
@@ -244,3 +254,70 @@ func validateNetworkingFlannel(v *kops.FlannelNetworkingSpec, fldPath *field.Pat
 
 	return allErrs
 }
+
+// validateELBSharedSecurityGroups tests if security groups are being used that loadbalancer security groups are used
+func validateELBSharedSecurityGroups(v *kops.LoadBalancerAccessSpec, securityGroups *kops.SecurityGroups, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	noLoadBalancerGroups := emptySlice(v.SecurityGroups)
+
+	// securityGroups are not defined and no load balancer security groups
+	// return no errors
+	if securityGroups == nil && noLoadBalancerGroups {
+		return allErrs
+	}
+
+	// securityGroups are defined and no load balancer security groups do not exist
+	if securityGroups != nil && noLoadBalancerGroups {
+		allErrs = append(allErrs, field.Required(fldPath.Child("SecurityGroups"), "SecurityGroups must be defined"))
```

Comment:
Can you confirm you correctly fall into this case when master/node SGs are defined but LB SGs are not? I tried to test it but ended up with a task error (although my setup may have been incorrect):
`error building tasks: unexpected error resolving task "SecurityGroupRule/https-elb-to-master": Unable to find task "SecurityGroup/masters.<cluster-name>", referenced from SecurityGroupRule/https-elb-to-master:.SecurityGroup`

Reply:
That is another bug that is fixed in the latest push.  But I do not thing validation is working fully :(